posthog/models/filters/mixins/common.py

```diff
@@ -42,9 +42,21 @@
 
 
 class IntervalMixin(BaseParamMixin):
+    """See https://clickhouse.tech/docs/en/sql-reference/data-types/special-data-types/interval/."""
+
+    SUPPORTED_INTERVAL_TYPES = ["second", "minute", "hour", "day", "week", "month", "quarter", "year"]
```

Comment:
Quarter and year exist here, but not in https://github.com/PostHog/posthog/pull/5467/files#diff-7622fa5a28c0ece9e0a59cc1e684e72252cfed2df3d57fa531b72705d7dda805R100 ? `PERIOD_TO_INTERVAL_FUNC`

Reply:
Good catch, I went for all the intervals supported by CH but they wouldn't pass validation anyway because of these `PERIOD_TO_*_FUNC` mappings. Fix #5484.